package.json

```diff
@@ -18,14 +18,18 @@
     "test": "gulp test",
     "test:watch": "gulp test-watch"
   },
+  "peerDependencies": {
+    "backbone": ">= 1.1.2",
+    "underscore": ">= 1.4.4"
+  },
   "dependencies": {
-    "backbone": "1.1.2",
+    "backbone": "1.0.0 - 1.1.2",
     "inflection": "^1.8.0",
     "jquery": "2.2.4",
-    "underscore": "1.4.4"
+    "underscore": ">= 1.4.4"
```

Comment:
We should remove this. We don't want this library to pull in its own copy of underscore if the host package has it. Right?

Reply:
Let's move them to `devDependencies`